library/src/main/java/com/felipecsl/gifimageview/library/GifDecoder.java

```diff
@@ -95,6 +95,8 @@
 
   private static final int INITIAL_FRAME_POINTER = -1;
 
+  static final int LOOP_FOREVER = 0;
```

Comment:
Hmm that makes sense, however I think you should explose a method `isLoopForever()` instead, that returns a boolean, instead of exposiing the `LOOP_FOREVER` constant